WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -428,6 +449,48 @@ - (NSString *)titleForHeaderInSection:(NSInteger)section
     return headingTitle;
 }
 
+- (void)showPrivacySelector
+{
+    NSArray *values = @[ @(BlogPrivacyPublic), @(BlogPrivacyHidden), @(BlogPrivacyPrivate)];
+    NSMutableArray *titles = [NSMutableArray array];
+    for (NSNumber * value in values) {
+        [titles addObject:[self.blog textForBlogPrivacy:[value integerValue]]];
+    }
+    NSArray *hints = @[
+                       NSLocalizedString(@"Your site is visible to everyone, and it maybe indexed in search engines.", @"Hint for users when public privacy setting is set"),
+                       NSLocalizedString(@"Your site is visible to everyone, but asks to search engine to not index your site", @"Hint for users when hidden privacy setting is set"),
```

Comment:
Small correction - `but asks to search engine to not` to `but asks search engines to not`. Also add a period at the end.